.github/workflows/firefox-ios-ui-tests.yml

```diff
@@ -190,7 +190,7 @@ jobs:
         with:
           payload-file-path: ${{ env.browser }}/slack.json
         env:
-          SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+          SLACK_WEBHOOK_URL: ${{ secrets.WEBHOOK_SLACK_TOKEN }}
```

Comment:
Just to double check, this workflow is only for the Smoketest?

Reply:
Yes, I encountered plenty of timeouts on the full functional tests.